crates/re_arrow_store/Cargo.toml

```diff
@@ -71,6 +71,7 @@ polars-ops = { workspace = true, optional = true, features = [
 
 
 [dev-dependencies]
+re_log_types = { workspace = true, features = ["testing"] }
```

Comment:
at the very least in this pr this doesn't seem to happen. Why is it needed now?
(also, makes me sad!)

Reply:
Probably needed at some point, dont remember... in any case it can never hurt and will probably save someone some pain in the future so I'll leave it there :shrug: